docs/requirements/requirements.md

```diff
@@ -0,0 +1,310 @@
+# Requirements
+
+:::caution
+
+This document is in the making since we are still in the requirements engineering phase.
+
+:::
+
+## RE Wording in this Document
+
+- We use *Microservice* and *Service* interchangeably.
+- We use *Reference Architecture* and *System* interchangeably.
+- RA = reference architecture
+
+## Stakeholders
+
+- Prof. Dr. Steffen Becker -- The Examiner
+- Sandro Speth -- Supervisor
+- Niklas Meißner -- Supervisor
+- The Developers
+- S -- Researcher
+- F -- Researcher
+- SH -- Researcher
+
+## Requirement Categories
+
+- Constraints
+- Process
+  - EnPro -- The project itself
+  - Software Artefacts
+- Product
+  - Domain
+  - Usability
+  - Non-functional Requirements & Quality Attributes
+  - Performance
+- Design
+  - Architecture
+  - Microservice Patterns
+  - Loose Coupling
+  - Technologies
+  - Deployment
+
+## Constraints
+
+### R-C-001: No external services
+
+- We are not allowed to use external services (e.g., external microservices).
+- We are not allowed to add functional dependencies to other services.
+- Origin: Supervisors
+
+### R-C-002: The use of the system must be free of charge
+
+- Deployment included.
+- Origin: Sarah
+
+## Process
+
+### EnPro Requirements
+
+#### R-E-001: 15 Microservices
+
+- The RA should consist of at least 15 microservices.
+- Origin: Supervisors
+
+#### R-E-002: Microservice Patterns
+
+- Microservice patterns must be used and well documented.
+- Origin: Supervisors
+
+#### R-E-003: Different Programming Languages
+
+- Different programming languages must be used for the system's implementation.
+- Python should only be considered as a last resort.
+  - If we used Python, we would have to explain why.
+
+- Origin: Examiner
+
+### Software Artefacts
+
+#### R-SArt-001: Good Documentation
+
+- A "good" documentation of the system is required.
+- The software architecture must be well documented.
+- When designing the system, we must document our design decisions using so called Architectural Design Records (ADRs).
+  - https://adr.github.io
+  - https://github.com/joelparkerhenderson/architecture-decision-record
+- Origin: Examiner
+
+#### R-SArt-002: Clean Code
+
+- Origin: Examiner
+
+#### R-SArt-003: Clean Deployment
+
+- The deployment must be "clean".
```

Comment:
what does this mean? do we have more information here? 
simple deployment was named multiple times, maybe we can add this here

Reply:
I added this as an open question for us to resolve later.